rules-reviewed/core.windup.label.xml

```diff
@@ -0,0 +1,144 @@
+                <tag>Stateful (SFSB)</tag>
+                <tag>Stateless (SLSB)</tag>
+                <tag>WS Metadata</tag>
+            </supported>
+            <unsuitable>
+                <tag>JAX-RPC</tag>
+                <tag>RMI</tag>
+            </unsuitable>
+            <neutral>
+                <tag>Bouncy Castle (embedded)</tag>
+                <tag>Java Source</tag>
+                <tag>JDBC datasources</tag>
+                <tag>JDBC XA datasources</tag>
+                <tag>Maven XML</tag>
+                <tag>Manifest</tag>
+                <tag>Swagger (embedded)</tag>
+                <tag>Spring (embedded)</tag>          <!-- Spring Core is supported, but Spring XML part of Spring Data Access is not supported-->
+                <tag>Spring MVC (embedded)</tag>      <!-- Spring MVC is supported -->
+                <tag>Spring Security (embedded)</tag> <!-- Spring Security is supported -->
+                <tag>Properties</tag>
+                <tag>Web XML*</tag>
+<!--                <tag>Web XML 3.0</tag>        &lt;!&ndash; temp &ndash;&gt;-->
+<!--                <tag>Web XML 3.1</tag>        &lt;!&ndash; temp &ndash;&gt;-->
+                <tag>WSDL (embedded)</tag>
+            </neutral>
+        </label>
+        <label id="jws">
+            <name>JWS</name>
+            <description>Red Hat Jboss Web Server</description>
+            <supported>
+                <tag>Bean Validation</tag>
+                <tag>JAXB</tag>
+                <tag>JSF Page</tag>
+                <tag>JSP Page</tag>
+                <tag>Servlet</tag>
+            </supported>
+            <unsuitable>
+                <tag>CDI</tag>
+                <tag>Clustering EJB</tag>
+                <tag>Clustering Web Session</tag>
+                <tag>Common Annotations</tag>
+                <tag>EAR</tag>
+                <tag>EJB</tag>
+                <tag>EJB XML*</tag>
+<!--                <tag>EJB XML</tag>-->
+<!--                <tag>EJB XML 3.1</tag>        &lt;!&ndash; temp &ndash;&gt;-->
+                <tag>Enterprise Web Services</tag>
+                <tag>JACC</tag>
+                <tag>Java EE Batch</tag>
+                <tag>Java EE Batch API</tag>
+                <tag>Java EE JSON-P</tag>
+                <tag>Java EE Security</tag>
+                <tag>JavaMail</tag>
+                <tag>JAX-RPC</tag>
+                <tag>JAX-RS</tag>
+                <tag>JAX-WS</tag>
+                <tag>JAXB</tag>
+                <tag>JAXR</tag>
+                <tag>JBoss EJB XML</tag>
+                <tag>JBoss Web XML</tag>
+                <tag>JCA</tag>
+                <tag>JMS</tag>
+                <tag>JMS Connection Factory</tag>
+                <tag>JMS Queue</tag>
+                <tag>JMS Topic</tag>
+                <tag>JPA</tag>
+                <tag>JPA entities</tag>
+                <tag>JPA named queries</tag>
+                <tag>JPA XML</tag>
+                <tag>JSON-B</tag>
```

Comment:
Is JSON-B unsupported?

Reply:
Ass agreed in our last meeting JSON-B will be supported by EAP and on JWS it won't be either supported, unsupported, or neutral